plugins/sonar-xoo-plugin/src/main/java/org/sonar/xoo/rule/XooRulesDefinition.java

```diff
@@ -52,6 +52,7 @@ public class XooRulesDefinition implements RulesDefinition {
 
   public static final String XOO_REPOSITORY = "xoo";
   public static final String XOO2_REPOSITORY = "xoo2";
+  private static final String CONSTANT_LITERAL="It takes about 1 minute to an experienced software craftsman to remove a line of code";
```

Comment:
The name should be more descriptive than CONSTANT_LITERAL

Reply:
Modified